src/utils/formatUserDetails.ts

```diff
@@ -9,27 +9,55 @@ export function convertTimeStamp(userDetails: UserResponseType) {
     const day = String(date.getDate()).padStart(2, "0");
     const month = String(date.getMonth() + 1).padStart(2, "0");
     const year = date.getFullYear();
+    if (flag) {
+      const monthName = date.toLocaleString("default", { month: "short" });
+      const formattedDate = `${day} ${monthName}, ${year}`;
+      return formattedDate;
+    }
     const formattedDate = `${day}/${month}/${year}`;
     return formattedDate;
   }
 
   return "N/A";
 }
+export function convertEpochToDate(timestamp: number | undefined) {
+  if (timestamp) {
+    const date = new Date(timestamp);
+
+    const day = String(date.getDate()).padStart(2, "0");
+    const month = date.toLocaleString("default", { month: "short" });
+    const year = date.getFullYear();
+    const formattedDate = `${day} ${month}, ${year}`;
+
+    return formattedDate;
+  }
+
+  return "N/A";
+}
 
 export function formatUserDetails(
   userDetails: UserResponseType,
   flag: boolean
 ) {
-  const convertedTimestamp = convertTimeStamp(userDetails);
-
-  const userId = `**User Id :** ${userDetails.user?.id}`;
-  const userName = `**User Name :** ${userDetails.user?.username}`;
-  const userFullName = `**Full Name :** ${userDetails.user?.first_name} ${userDetails.user?.last_name}`;
-  const userState = `**State :** ${userDetails.user?.state}`;
-  const discordJoinedAt = `**Joined Server on :** ${convertedTimestamp}`;
+  const convertedTimestamp = convertTimeStamp(userDetails, flag);
+  const accountCreationDate = convertEpochToDate(userDetails.user?.created_at);
+
+  const userId = " ".repeat(30) + `**Id:** ${userDetails.user?.id}`;
```

Comment:
Why create a " " of length 30?

Reply:
spaces have been added to format all the user details correctly, and for aligning all the colons in a straight line